src/crc32c_arm64.cc

```diff
@@ -29,14 +30,15 @@
 // compute 8bytes for each segment parallelly
 #define CRC32C32BYTES(P, IND)                                             \
   do {                                                                    \
-    crc1 = __crc32cd(                                                     \
-        crc1, *((const uint64_t *)(P) + (SEGMENTBYTES / 8) * 1 + (IND))); \
-    crc2 = __crc32cd(                                                     \
-        crc2, *((const uint64_t *)(P) + (SEGMENTBYTES / 8) * 2 + (IND))); \
-    crc3 = __crc32cd(                                                     \
-        crc3, *((const uint64_t *)(P) + (SEGMENTBYTES / 8) * 3 + (IND))); \
-    crc0 = __crc32cd(                                                     \
-        crc0, *((const uint64_t *)(P) + (SEGMENTBYTES / 8) * 0 + (IND))); \
+    uint64_t d;                                                           \
+    std::memcpy(&d, (P) + SEGMENTBYTES * 1 + (IND) * 8, sizeof d);        \
```

Comment:
Nit: though not required, we usually use `sizeof(d)`. Mind updating to include the parentheses for all of these?

Reply:
Thanks! Updated.